### PR TITLE
docs(btp,i18n): add license link in i18n and btp packages' README.md

### LINKED ISSUE #11900: i18n & btp NPM license question
Regarding https://www.npmjs.com/package/@fundamental-ngx/i18n and https://www.npmjs.com/package/@fundamental-ngx/btp, is there a license that applies to these two NPM packages? I noticed that the other fundamental-ngx scoped packages have license and repo information but these two do not.

### PR DESCRIPTION
**Pull request by droshev**
fixes #11900

### CODE REVIEW

**Review by InnaAtanasova — CHANGES_REQUESTED** [review]

File: libs/btp/README.md [hunk 1]
```diff
@@ -1,7 +1,76 @@
...
+-   [7. Contributing](#7)
+-   [8. License](https://github.com/SAP/fundamental-ngx/blob/main/LICENSE.txt)
+-   [9. Similar Projects](#8)
+
+## <a name="1"></a>1. Description
+
+The `@fundamental-ngx/btp` library is a set of utilities and [Angular](https://angular.io/) directives aimed to help developers build custom UI components with common interaction patterns.
+
+It includes common behaviour for working with `DataSources` and `ControlValueAccessors`.
+Additionally, it contains injectable helpers to easily control the `rxjs` subscriptions in your components, focusable/selectable list helpers and many more.
```
[the remark this answers is not in the document]
Comment (droshev) [reply libs/btp/README.md]: pushed an update

File: libs/btp/README.md [hunk 2]
```diff
@@ -1,7 +1,76 @@
...
+-   [4. Getting Started](#4)
+-   [5. Known Issues](#5)
+-   [6. Support](#6)
+-   [7. Contributing](#7)
+-   [8. License](https://github.com/SAP/fundamental-ngx/blob/main/LICENSE.txt)
+-   [9. Similar Projects](#8)
+
+## <a name="1"></a>1. Description
+
+The `@fundamental-ngx/btp` library is a set of utilities and [Angular](https://angular.io/) directives aimed to help developers build custom UI components with common interaction patterns.
```
[the remark this answers is not in the document]
Comment (droshev) [reply libs/btp/README.md]: pushed an update

File: libs/btp/README.md [hunk 3]
```diff
@@ -1,7 +1,76 @@
...
+
+It includes common behaviour for working with `DataSources` and `ControlValueAccessors`.
+Additionally, it contains injectable helpers to easily control the `rxjs` subscriptions in your components, focusable/selectable list helpers and many more.
+
+See [Component Documentation](https://sap.github.io/fundamental-ngx/#/core/new-component) for examples and API details.
+
+## <a name="2"></a>2. Requirements
+
+To download and use Fundamental Library for Angular, you will first need to install the [node package manager](https://www.npmjs.com/get-npm).
```
[the remark this answers is not in the document]
Comment (droshev) [reply libs/btp/README.md:27]: pushed an update

**Comment by droshev** [comment]
@InnaAtanasova Thank you for the review, you made very good points. I have updated the file

**Review by InnaAtanasova — APPROVED** [review]

**Pull request merged**